pkg/agent/sysadvisor/plugin/poweraware/evictor/load_evictor.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2022 The Katalyst Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package evictor
+
+import (
+	"context"
+
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/kubewharf/katalyst-core/pkg/config/generic"
+	"github.com/kubewharf/katalyst-core/pkg/metaserver/agent/pod"
+	"github.com/kubewharf/katalyst-core/pkg/util/general"
+)
+
+// PercentageEvictor is the interface used in advisor policy
+type PercentageEvictor interface {
```

Comment:
Let's also rename the file name.

Reply:
will rename file to percentage_evictor.go & percentage_evictor_test.go